dbcon/mysql/is_columnstore_tables.cpp

```diff
@@ -142,7 +142,7 @@ static int is_columnstore_tables_fill(THD* thd, TABLE_LIST* tables, COND* cond)
         if (tb_info.tablewithautoincr)
         {
             table->field[5]->set_notnull();
-            table->field[5]->store(systemCatalogPtr->nextAutoIncrValue((*it).second));
+            table->field[5]->store(systemCatalogPtr->nextAutoIncrValue((*it).second), lower_case_table_names);
```

Comment:
`lower_case_table_names` should be passed as an argument to nextAutoIncrValue(). I am not sure how this is compiling.

Reply:
It's defaulted. However, I added it in because, while I expect tablename to always be lower case here because it's a system catalog name, and we always create those lower case, there's no guarantee of that in the future.

Reply:
Changed again. It seems information schema should ignore the lower_case_table_names